### ISSUE TITLE
[FEATURE REQUEST] Have an option to have the clock on the left to the right of the weather or force the weather to the right of the weather

### ISSUE THREAD

**Issue by albiek1976** [martinet101/ElevenClock#1055]
**This isn't really a problem but more of a suggestion**
**NOTE:** _This is similar to #912 but I don't believe is the same thing as being requested_

The clock on the left now is overlaying of the weather on Windows 11. 
I would like to **use** on my computer the dual mode or clock on the left more often or permanently. 

I would like to see a user control repositioning the clock on the left-screen side
**The weather and clock could swap positions and both be seen at the same time.  
The user could also have the choice as of now with the weather not to be seen and just to have the clock shown.**

**Comment by martinet101** [comment]
you can do this, if you do the following:

Note that the weather widget can't be moved, but what you can do is to adjust the clock horizontal position offset so it shows on the side.

you can change this from the settings window  under clock position

**Comment by albiek1976** [comment]
**This is issue #1.** 

I would like dual clocks on at all times and the clock on the right to have the settings maintained

The issue with that is I want to keep the clock on the right locked in the same position set with changes.
On the primary screen adjust the horizontal The clock on the right ends up in that position in the picture.
On the second and tertiary screens, the clock on the right disappears. 

<img width="1919" alt="Screenshot 2022-12-04 110925 (2)" src="https://user-images.githubusercontent.com/92185158/205503013-f396cc30-9aad-465a-9d67-4fe040dff18c.png">





**To sum this up, Issue #1.  
Separate left and right clock position horizontal controls would solve the issue with dual clocks and the clock on the right disappearing on screens #2 & 3**


<img width="824" alt="Horizontal" src="https://user-images.githubusercontent.com/92185158/205502855-9497e94c-74f9-4041-baeb-f1af07602d57.png">

**Comment by martinet101** [comment]
> **NOTE:** _This is similar to #912 but I don't believe is the same thing as being requested_

Yes, the fix is actually the same on #912. Please oin that thread to stay tuned about further updates
